libs/client-sdk/README.md

```diff
@@ -8,6 +8,8 @@ The LaunchDarkly Client-Side SDK for C/C++ is designed primarily for use in desk
 It follows the client-side LaunchDarkly model for single-user contexts (much like our mobile or JavaScript SDKs). It is
 not intended for use in multi-user systems such as web servers and applications.
 
```

Comment:
TODO: The readme currently lists the cmake variables used to control the build. Should we move this into the top-level README, or copy into the server-sdk's README?


 Right now they are all general rather than sdk-specific.